apps/examples/testcase/le_tc/kernel/tc_mqueue.c

```diff
@@ -16,41 +16,9 @@
  *
  ****************************************************************************/
 
-/// @file mqueue.c
+/// @file tc_mqueue.c
+
 /// @brief Test Case Example for Message Queue API
-/**************************************************************************
```

Comment:
This tc was come from mqueue.c at kernel_sample and kernel_sample was renamed from ostest of Nuttx. This is almost same as mqueue of kernel_sample. We should not remove Nuttx license.

Reply:
yes, Nuttx license has to be there I will revert it